### PR TITLE
ports cornu cutaneum viro symptom (Port)

### PR DESCRIPTION
**Pull request by boodaliboo**
original pr https://github.com/BeeStation/BeeStation-Hornet/pull/1468
"The virus causes the host to unpredictably grow and shed sharp spines, damaging those near them."
Transmission 6: Gives the host some armor against brute damage.
Resistance 6: Spikes grow faster and hurt you more often

#### Changelog

:cl:  
rscadd: Added new spike virus
/:cl:

### CODE REVIEW

**Comment by drderp3635** [comment]
this seems like it would be better in genetics than virology

File: code/datums/diseases/advance/symptoms/spiked.dm [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	stealth = -3
+	resistance = 3
+	stage_speed = -3
+	transmittable = 0
+	level = 0
+	symptom_delay_min = 1
+	symptom_delay_max = 1
+	severity = 1
+	base_message_chance = 5
+	var/Power = 1
```
Comment (JamieD1) [inline code/datums/diseases/advance/symptoms/spiked.dm]: why is this a capital?

File: code/datums/diseases/advance/symptoms/spiked.dm [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+	var/get_damage = 0
+	var/ closrf = 0
```
Comment (JamieD1) [inline code/datums/diseases/advance/symptoms/spiked.dm]: space?

File: code/datums/diseases/advance/symptoms/spiked.dm [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+/datum/symptom/spiked/Activate(var/datum/disease/advance/A)
+	var/mob/living/M = A.affected_mob
+	var/mob/living/carbon/human/H = A.affected_mob
+
+	if(!..())
+		return
+
+	if(prob(20*power))
+		closrf = 1
+	else
```
Comment (JamieD1) [inline code/datums/diseases/advance/symptoms/spiked.dm:55]: not needed remove the else as its already 0 by default

File: code/datums/diseases/advance/symptoms/spiked.dm (hunk 2 above)
Comment (JamieD1) [reply code/datums/diseases/advance/symptoms/spiked.dm]: Did this even compile?

**Review by JamieD1 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**